release.sh

```diff
@@ -154,6 +154,8 @@ while getopts ":celLzusop:dw:a:r:t:g:m:" opt; do
 		# shortcut for classic
 		if [ "$OPTARG" = "classic" ]; then
 			game_type="classic"
+		elif [ "$OPTARG" = "bc" ]; then
+			game_type="bc"
 			# game_version from toc
 		else
 			# Set version (x.y.z)
```

Comment:
The block following this can set the game_type based on the TOC version and should handle BC versions as well.